api/descriptions/users/post.md

```diff
@@ -1,10 +1,36 @@
 Adds a new user to the system.
 
+To invoke this endpoint in the Console UI:
+
+1. Navigate to **Manage > Authentication > Users**.
+2. Click **+ Add user** and enter the user's information.
+3. Click the **Save** button.
+
+### cURL Request
+
+The following cURL command adds a new user.
+
 ```bash
-$ curl -k \
+$ curl 'https://<CONSOLE>/api/v1/users' \
```

Comment:
@KICKdesigns Great job exposing the `permissions` object.

I'll have some more feedback here shortly. I've pinged my team internally to see what other use cases we should document.

In the meantime, here's my feedback:

1. Let's present multiple curl commands here, starting with the most basic one. In the most basic form, the `permissions` object isn't required (let me know if you discovered otherwise):

   ```
   curl 'https://<CONSOLE>/api/v1/users' \
   -k \
   -X POST \
   -u <USER> \
   -H 'Content-Type: application/json' \
   -d \
   '{
    "username":"<USER_NAME>",
    "password":"<PASSWORD>",
    "role":"auditor",
    "authType":"basic"    
   }'
   ```

2. Let's add a second example that uses the permissions object. There are two main states: Projects are disabled (default) or projects are enabled. Let's discuss the full permissions object (with the projects field), but let's craft an example that just assigns collections.

Ping me if you've got any questions.